src/please.go

```diff
@@ -72,7 +72,7 @@ var opts struct {
 
 	FeatureFlags struct {
```

Comment:
This is confusingly named. Feature flags are what I'm calling the things that gate breaking changes between releases. Might want to revisit this.

Reply:
This was there first :-P

But agreed. It's only internal to this package so we can call it whatever we want.

Reply:
Oh okay, I assumed this affected `plz halp`

Reply:
This is the struct that defines the "Options that enable / disable certain features" section in `plz --help`, but the actual name of the struct doesn't appear there.